.stylelintignore

```diff
@@ -1 +1,2 @@
 media/css/libs/*
+media/css/externalpages/pocket/*
```

Comment:
I found a better way to do this that solves the issue you we're describing on Slack. If you delete this `.stylelintignore` file altogether, and instead add the following to `.stylelintrc` then it should play nice with the pre-commit hook:

```json
"ignoreFiles": [
    "media/css/libs/**/*",
    "media/css/externalpages/pocket/**/*"
]
```